sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/evaluate/PodInfoBuilder.java

```diff
@@ -315,7 +388,7 @@ private static void extendEnv(Protos.CommandInfo.Builder builder, Map<String, St
         return builder.build();
     }
 
-    private static void setHealthCheck(
+    private void setHealthCheck(
```

Comment:
I favor them being non-static -- the need to have the same data passed in to several of these static methods indicates to me that they are in fact functions of the object and not the class.